ite8291r3_ctl/__main__.py

```diff
@@ -332,6 +350,7 @@ def handle_query_args(args):
 	parser_mode = subparsers.add_parser('mode', help='Enable interactive modes.')
 	group = parser_mode.add_mutually_exclusive_group()
 	group.add_argument('--screen', metavar='offset_x,offset_y,width,height', nargs='?', const='fullscreen', help='Color the keyboard according to what is on the screen in the given region.')
+	group.add_argument('--image', metavar='image_path', nargs='?', const='fullscreen', help='Color the keyboard according to a given image.')
```

Comment:
```suggestion
	group.add_argument('--image', metavar='path', nargs=1, type=argparse.FileType('rb'), help='Color the keyboard according to a given image.')
```

Or something like this.